### DISCUSSION TITLE
Unable to log into GitHub. How long does a ip ban last?

### DISCUSSION THREAD

**Discussion by LavajavaLux** [community/community#82784] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Feature Area

Issues

### Body

I can’t log into my GitHub account. I changed my password, then logged in with that password, it worked. But when I tried on another device it didn’t and now it says my ip is blocked and I can’t even log in on my device where it has worked. How long does this ip ban last? I’m only able to write this because I didn’t close the window from when I first tried my new password

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by Ashishdheer** [comment, marked as answer]
GitHub can ban an IP address for a period of time if the IP repeatedly fails to log in. The duration of the ban lasts for a maximum of 3 days or even less. So, make sure to check at least after a day.
